workflow.txt

```diff
@@ -1,10 +1,13 @@
-git clone
+fork on github
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
+git clone
```

Comment:
You need to clone before you `checkout`. Otherwise, there's no repo in which to `checkout`.